app/controllers/application_controller.rb

```diff
@@ -1,2 +1,7 @@
 class ApplicationController < ActionController::Base
-end
+  def x e
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Method parameter must be at least 3 characters long.